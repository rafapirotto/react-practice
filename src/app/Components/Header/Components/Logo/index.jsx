import React from 'react';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';

import styles from './styles/Logo.module.css';

const propTypes = {
  to: PropTypes.string.isRequired,
};

const Logo = ({ to }) => (
  <Link to={to}>
    <div id="logo-btn" className={`text-4xl ${styles.movy}`}>
      Movy
    </div>
  </Link>
);

Logo.propTypes = propTypes;

export default Logo;
