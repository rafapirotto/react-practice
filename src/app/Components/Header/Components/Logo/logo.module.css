.movy {
  font-family: 'Exo', sans-serif;
}
