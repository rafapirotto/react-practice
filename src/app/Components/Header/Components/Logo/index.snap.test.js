import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import Logo from '.';

it('matches the snapshot', () => {
  const component = renderer.create(
    <Router>
      <Logo />
    </Router>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
