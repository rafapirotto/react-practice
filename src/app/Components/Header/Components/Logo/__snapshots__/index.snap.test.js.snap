// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  <div
    className="movy text-4xl"
  >
    Movy
  </div>
</a>
`;
